features/google/src/model/importers/activities-importer.js

```diff
@@ -43,14 +43,14 @@ class ActivityParser {
 }
 
 export default class ActivitiesImporter {
-    async import({ zipFile, googleAccount }) {
+    async import({ zipFile, facebookAccount }) {
```

Comment:
Did we actually hard code `facebookAccount` in _poly-import_? I thought it was called `dataAccount` there? Anyway, very suspicious change - if it's necessary so be it, I guess... To signify that this is more of a workaround here you could do a restructure rename I guess: `async import({ zipFile, facebookAccount: googleAccount })` - just in case it makes more sense to fix that later.

Reply:
Yeah discovered that at this point it is actually still facebookAccount. Wanted to fix this asap but in another PR. Yet I like your suggestion to show it's just a workareound until the other PR is merged